tests/direct_adaptor_factory_wrapper.py

```diff
@@ -0,0 +1,133 @@
+import json
+import logging
+import time
+import sys
+import utils.utility as enclave_helper
+import utils.signature as signature
+from service_client.generic import GenericServiceClient
+import workorder_json
+from tcf_connector.direct_json_rpc_api_adaptor_factory import DirectJsonRpcApiAdaptorFactory
+import utils.hex_utils as hex_utils
+
+# Remove duplicate loggers
+for handler in logging.root.handlers[:]:
```

Comment:
Why do we need this?

Reply:
Without it the logger displays all messages twice with slightly different formatting. I think it is caused by 
 `logging.basicConfig(format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO)`
being used in the tcf_connector and ethereum adaptors instead of using getLogger as in most other files but I have not been able to change all such files yet in order to check if it would fix the issue.